src/config.js

```diff
@@ -20,6 +21,12 @@ const config: Config = {
   // better off giving up.
   requestLongTimeoutMs: 60 * 1000,
 
+  // The topic recognized by servers as "there is no topic". This should
+  // match
+  //   https://github.com/zulip/zulip/blob/5.0-dev/zerver/lib/actions.py#L2714
+  // or similar logic at the latest `main`.
+  noTopicTopic: '(no topic)',
```

Comment:
Centralizing a single definition for this constant makes a lot of sense!

I don't love "config" as a home for it. It's not really so much about a choice of how to configure the app, as it is about something we expect from the server.

Hmm, I guess in fact that suggests a good place for it would be under `src/api/` somewhere -- that's where we generally put facts about how the Zulip server behaves. Usually those are in the form of code to invoke, or type definitions, but a constant like this would fit too.

I think a file like `src/api/constants.js`, or `src/api/apiConstants.js`, would work well.

Reply:
Oh, and https://github.com/zulip/zulip/issues/3639 is the issue for the fact that this is hardcoded in the server (and therefore untranslated.)